### PR TITLE
Fixing unresolved link to API docs.

### PR DESCRIPTION
**Pull request by Serdaro**

### CODE REVIEW

**Comment by asimarslan** [comment]
verify

**Comment by asimarslan** [comment]
verify

**Comment by asimarslan** [comment]
run-net

**Comment by asimarslan** [comment]
run-net

**Comment by asimarslan** [comment]
runnet

**Comment by asimarslan** [comment]
verify

**Comment by stalek71** [comment]
verify

File: README.md [hunk 1]
```diff
@@ -588,7 +588,7 @@ cfg.GetNetworkConfig().AddAddress("127.0.0.1:5701");
 var client = HazelcastClient.NewHazelcastClient(cfg);
 ```
 
-See the `ClientConfig` class documentation at [Hazelcast .NET client API Docs](http://docs.hazelcast.org/docs/clients/net/current/) for details.
+See the `ClientConfig` class documentation at [Hazelcast .NET client API Docs](https://docs.hazelcast.org/docs/clients/net/3.10.0/html/G_Hazelcast_Client.htm) for details.
```
Comment (asimarslan) [inline README.md]: correct link should be :
https://docs.hazelcast.org/docs/clients/net/latest/
instead of a fixed version in the link

**Comment by zpqrtbnk** [comment]
Hey. Stephan here. We are about to release version 4.0 of the Hazelcast.Net client, and our README and documentation have completely been overhauled, making this PR... unfortunately irrelevant. Sorry for not merging earlier, there has been a period where we were not paying sufficient attention to PRs, and we want to be better at it in the future. I want to thank you for your contribution, and hope to see you again here!

**Pull request closed without merging**